dotCMS/src/main/java/com/dotmarketing/util/starter/ExportStarterUtil.java

```diff
@@ -354,8 +391,236 @@ private void createJSONFiles()  {
             throw new DotRuntimeException(e);
         }
     }
-    
-    
+
+    /**
+     * Takes all the data structures in dotCMS that are required for the system to start up, and creates an in-memory
+     * JSON representation of them. No assets are included in this process.
+     * <p>For Hibernate classes and classes that map to database tables, the {@link DotSubmitter} API is used to
+     * generate the required JSON files in parallel in order to speed up this process as much as possible.</p>
+     *
+     * @return The list of {@link FileEntry} objects containing the required information of all the required data
+     * structures in dotCMS.
+     */
+    public List<FileEntry> getStarterDataAsJSON()  {
+        Logger.info(this, "Converting all repository data to JSON data...");
+        final List<FileEntry> starterFiles = new ArrayList<>();
+        final Set<Class<?>> dotcmsTablesToDump = Try.of(() -> this.getTableSet()).getOrElse(new HashSet<>());
+        final List<List<Class<?>>> dotcmsTablesSubset = this.splitTableList(dotcmsTablesToDump, 10);
+        try {
+            final DotSubmitter dotSubmitter = DotConcurrentFactory.getInstance().getSubmitter("starter_export_submitter",
+                    new DotConcurrentFactory.SubmitterConfigBuilder().poolSize(10).maxPoolSize(40).queueCapacity(1000).build());
```

Comment:
Make the pool sizes/queue configurable.

Reply:
yeah, do not forgot to use static final or lazy's

Reply:
Done!